docs/src/pages/docs/functions/combinators.md

```diff
@@ -20,6 +20,32 @@ give it a value and it will give you back a function ready to take a function.
 Once that function is provided, it will return the result of applying your value
 to that function.
 
+```javascript
+import applyTo from 'crocks/combinators/applyTo'
+
+import map from 'crocks/pointfree/map'
+import compose from 'crocks/helpers/compose'
+
+// prices :: [ Number ]
+const prices = [4.99, 29.99, 15.99];
+
+// getPrices :: (a -> b) -> [ Number ]
+const getPrices = compose(
+  applyTo(prices),
+  map
+);
+
+// discount :: Number -> Number -> Number
+const discount = percent => price =>
+  Number((price - (percent / 100) * price).toFixed(2));
```

Comment:
Same thing here with the semi-colon

Reply:
Yeah going to work with Karth to figure out how this got past the checks

Reply:
Okay, looks like I found the issue. Husky has some troubles finding the right git on a Mac.

```
husky > Setting up git hooks
Husky requires Git >= 2.13.2, please update Git
husky > Failed to install
```